src/tasks/DailyTask.py

```
@@ -458,44 +458,76 @@ def gongongqu(self):
 
+    def _auto_loop_step_with_retry(self, step_num, match, box=None, time_out=5, after_sleep=2, **kwargs):
+        """
+        点击OCR匹配元素，等待点击特征在 3s 内消失。
+        若特征未消失则重试，最多尝试 3 次。
+        """
+        for attempt in range(3):
+            result = self.wait_click_ocr(
+                match=match, box=box, time_out=time_out, after_sleep=0, log=True, **kwargs
+            )
+            if not result:
+                self.log_info(f"自主循环步骤{step_num}未完成，退出循环", notify=True)
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)